tests/virtual_machines_test.go

```diff
@@ -403,6 +403,12 @@ var _ = Describe("Virtual Machines", func() {
 		})
 		//2633
 		Context("When we re-apply a failed VM yaml", func() {
+			totalTimeout := time.Duration(0)
+			BeforeAll(func() {
+				vmFailCleanupWaitTime, err := getVmFailCleanupWaitTime()
+				Expect(err).ToNot(HaveOccurred(), "Should succeed getting wait-time argument")
```

Comment:
Can you change it like you did [here](https://github.com/k8snetworkplumbingwg/kubemacpool/pull/152#discussion_r429628949)? :slightly_smiling_face: